code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -330,12 +330,28 @@ GLOBAL_LIST_INIT(cloth_recipes, list ( \
 
 /obj/item/stack/sheet/cloth/five
 	amount = 5
+	
+/obj/item/stack/sheet/wool
+	name = "wool"
+	desc = "A bundle of wool ready to be spun on the loom."
+	singular_name = "wool ball"
+	icon_state = "sheet-wool"
+	item_state = "sheet-wool"
+	icon = 'icons/obj/stack_objects.dmi'
```

Comment:
This line is unnecessary
```suggestion
```

Reply:
I didn't have it before, I had to add it otherwise the icon wouldn't load for some reason